pkg/cmd/apply.go

```
@@ -373,6 +397,37 @@ func runApplyUpgradeIfNeeded(s *state.State, opts *applyOpts) error {
 	return errors.Wrap(tasksToRun.Run(s), "failed to reconcile the cluster")
 }
 
+func runApplyRotateKey(s *state.State, opts *applyOpts) error {
+	if !opts.ForceUpgrade {
+		s.Logger.Error("rotating encryption keys requires the --force-upgrade flag")
+		return nil
```

Shouldn't we return an error here?